mapviz_plugins/CMakeLists.txt

```diff
@@ -209,13 +209,13 @@ target_link_libraries(${PROJECT_NAME} PUBLIC
     tf2_geometry_msgs::tf2_geometry_msgs
     tf2_ros::tf2_ros
     ${visualization_msgs_TARGETS})
-target_include_directories(${PROJECT_NAME} INTERFACE 
+target_include_directories(${PROJECT_NAME} INTERFACE
   ${swri_roscpp_TARGETS}
 )
 
 ### Install the plugins ###
 install(DIRECTORY include/
-  DESTINATION include/${PROJECT_NAME}
+  DESTINATION include/
```

Comment:
I really don't think we should change this. @DangitBen what are your thoughts here?

Reply:
I might be right but for the wrong reasons? Or maybe both are true? There's this page that explains some rationale:
https://colcon.readthedocs.io/en/released/user/overriding-packages.html#install-headers-to-a-unique-include-directory